core/domain/exp_domain.py

```diff
@@ -276,8 +278,13 @@ class ExplorationChange(change_domain.BaseChange):
     }, {
         'name': CMD_ADD_TRANSLATION,
         'required_attribute_names': [
-            'state_name', 'content_id', 'language_code', 'content_html',
-            'translation_html'],
+            'state_name', 'content_id', 'language_code',
+            'needs_update', 'translation_html', 'data_format'],
+        'optional_attribute_names': ['content_html'],
```

Comment:
`content_html` should be a mandatory field for translation suggestions change cmd, maybe create a separate command for suggestions and exploration_editor page? [This will help us to update them in the future without worrying about other being cont compatible, wdyt?]

Reply:
Can you please explain why 'content_html' needs to be mandatory for translation suggestions e.g. where it will be used? I did not find it being used in the relevant if block for the ADD_TRANSLATION cmd and hence removed it.

Reply:
Created a new commit command.